test/functional/specs/desktop/C8118.js

```diff
@@ -0,0 +1,32 @@
+import { t, Selector } from "testcafe";
+import createNetworkLogger from "../../src/networkLogger";
+import { responseStatus } from "../../src/assertions/index";
+import testServerUrl from "../../src/constants/testServerUrl";
+
+const linkPageWithClickHandler = `${testServerUrl}/test/functional/sandbox/html/linkPageWithClickHandler.html`;
+
+const networkLogger = createNetworkLogger();
+
+fixture`C8118`
+  .page(linkPageWithClickHandler)
+  .requestHooks(
+    networkLogger.gatewayEndpointLogs,
+    networkLogger.sandboxEndpointLogs
+  );
+
+test.meta({
+  ID: "C8118",
+  SEVERITY: "P0",
+  TEST_RUN: "Regression"
+});
+
+test("Regression: Load page with link. Click link. Verify request.", async () => {
+  await t.click(
+    Selector("#body > section > div.left-nav > div > ul > li:nth-child(1) > a")
```

Comment:
I would add a data attribute or something on that element instead, just in case the html changes which will break the test.

Reply:
Done